pwiz_tools/Skyline/Model/Tools/ToolDescription.cs

```diff
@@ -887,15 +887,30 @@ public static void GetReport(SrmDocument doc, string reportTitle, string toolTit
             }
         }
 
+

```

Comment:
Did the abbreviation end up solving a problem? If not then I personally would rather have the full test names (possibly without the redundant "Test" prefix or suffix). It also seems like it could fairly easy collide where the full versions would be unique: TestMyCoolFunc and TestMyCoolFeat would both abbreviate the same, right?

Reply:
It did, yes. An unzip operation failed because it was trying to unzip to a deeply nested directory under the java install, which already had quite a long path due to the length of the test name.

TestMyCoolFunc and TestMyCoolFeat would indeed abbreviate the same, but that's a bridge I'm willing to jump off when I come to it.

Fundamentally I'd rather just fix Skyline so it doesn't care about path lengths - I believe it's possible but I haven't look into it.